### PR TITLE
test: refactor using `.config.js` (JS) for test configs

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: test/configs/sourceMap-true.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  file: 'sourceMap-true',
```
Comment (michael-ciniawsky) [inline test/configs/sourceMap-true.config.js]: `sourceMap-true` => `sourceMap`

File: test/configs/plugins-function.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+test('Plugins - function', () => {
```
Comment (michael-ciniawsky) [inline test/configs/plugins-function.js]: `function` => `{Function}`

File: test/configs/plugins-function.config.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  file: 'plugins-function',
```
Comment (michael-ciniawsky) [inline test/configs/plugins-function.config.js]: `plugins-function` => `plugins.fn || pluginsFn`

File: test/configs/sourceMap-true.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-test('Source Maps', () => {
+test('Source Maps - true', () => {
```
Comment (michael-ciniawsky) [inline test/configs/sourceMap-true.js]:
```diff
- true
```

File: test/configs/sourceMap-inline.config.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  file: 'sourceMap-inline',
```
Comment (michael-ciniawsky) [inline test/configs/sourceMap-inline.config.js]: `sourceMap-inline` => `map` ( Referring to PostCSS `{ map: 'inline'}`)

File: test/configs/sourceMap-inline.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+test('Source Maps - inline', () => {
```
Comment (michael-ciniawsky) [inline test/configs/sourceMap-inline.js]: `inline` => `Inline`

File: test/configs/sourceMap-true.config.js (hunk 1 above)
Comment (evilebottnawi) [inline test/configs/sourceMap-true.config.js]: @michael-ciniawsky i think we late add `sourceMap - false`

File: test/configs/sourceMap-true.js (hunk 4 above)
Comment (evilebottnawi) [inline test/configs/sourceMap-true.js]: @michael-ciniawsky https://github.com/postcss/postcss-loader/pull/230#discussion_r115759553

File: test/configs/sourceMap-inline.config.js (hunk 5 above)
Comment (evilebottnawi) [inline test/configs/sourceMap-inline.config.js]: @michael-ciniawsky Will not it be a bit confused?

Comment (michael-ciniawsky) [inline test/configs/sourceMap-inline.config.js]: yeah... 🙃 😛  I just don't like the `dash` style :D

**Comment by michael-ciniawsky** [comment]
😕 😛

**Pull request merged**